### ISSUE TITLE
Tag pages do not work

### ISSUE THREAD

**Issue by undr22** [vividvilla/ezhil#60]
baseURL/tags can be generated correctly.

But no matter which tag I click, they all link to an "All tags" page like this(but with correct URL):

<img width="400" alt="屏幕快照 2020-10-22 上午5 18 26" src="https://user-images.githubusercontent.com/55726154/96788390-05969700-1426-11eb-80be-0c4300758344.png">

And I noticed that [the demo on themes.gohugo.io](https://themes.gohugo.io//theme/ezhil/tags) also have this problem.

Hugo version v0.76.5/extended darwin/amd64.

**Comment by ehmatthes** [comment]
This pull request seems to fix the issue: <https://github.com/vividvilla/ezhil/pull/59>

TLDR; delete the term.html layout in ezhil/layouts/_default, and the site will use the default Hugo template for the tag page. At least that's how I'm understanding this fix.

**Comment by BDHU** [comment]
Can confirm this. I'm using v0.82.0-9D960784/extended on Ubuntu 20.04 and it happens to me as well.

**Comment by YulongNiu** [comment]
> This pull request seems to fix the issue: #59
> 
> TLDR; delete the term.html layout in ezhil/layouts/_default, and the site will use the default Hugo template for the tag page. At least that's how I'm understanding this fix.

It works.
